src/test/Components/CreateImageWizard/steps/Snapshot/Snapshot.test.tsx

```diff
@@ -286,6 +302,23 @@ describe('repository snapshot tab - ', () => {
     await clickRevisitButton();
     await screen.findByRole('heading', { name: /Custom repositories/i });
   });
+
+  test('select use a content template', async () => {
+    await renderCreateMode();
+    await goToSnapshotStep();
+    await selectUseTemplate();
+    const nextBtn = await screen.findByRole('button', { name: /Next/i });
```

Comment:
We've got a `getNextButton` helper function for this in `wizardTestUtils.tsx`, so you could just do:
```suggestion
    const nextBtn = await getNextButton();
```
but this is super minor nitpick, so feel free to ignore :sweat_smile:

Reply:
ah woops missed that, thanks 🫶